### PR TITLE
Copyedit Rmd files & rebuild book

### PR DESCRIPTION
**Pull request by pkq**
As promised months ago, I finally got around to re-reading and suggesting some minor enhancements to this already fantastic resource. I know you said much of this content has been superseded by Modern Dive, but hopefully some of it is still helpful to somebody. Happy to do something similar for the latter as well, if you're so inclined. No obligation to merge any of this, of course. Let me know what you think.

### CODE REVIEW

File: 07-conclusion.Rmd [hunk 1]
```diff
@@ -1,27 +1,11 @@
...
 
-Of course, this book is just the tip of the iceberg in terms of showing you what R
-can really do.  If you'd like to learn more I encourage you to check out Albert Kim
-and I's online, free, open-source book on using modern data analysis techniques
-and visualization with R, RStudio, and R Markdown.  More details on that project
-will be coming soon.
+Of course, this book is just the tip of the iceberg in terms of showing you what R can really do. If you'd like to learn more, I encourage you to check out [Modern Dive](https://ismayc.github.io/moderndiver-book/), a free, online, open-source book Albert Kim and I wrote on using modern data analysis techniques and visualization with R, RStudio, and R Markdown.
```
Comment (ismayc) [inline 07-conclusion.Rmd]: @pkq Can you switch this to https://www.moderndive.com instead of my GitHub?

File: index.Rmd [hunk 2]
```diff
@@ -1,4 +1,4 @@
---- 
+---
 title: "Getting used to R, RStudio, and R Markdown"
 author: "Chester Ismay"
```
Comment (ismayc) [inline index.Rmd]: @pkq Please add yourself as co-author or editor here. I'm sure this took a ton of time and I'm very appreciative of the effort!

**Review by ismayc — CHANGES_REQUESTED** [review]
@pkq If you could address my two requests and do a rebuild, I'd be happy to get this merged. It also deploys to https://rbasics.netlify.com/ as well from the `gh-pages` branch.

File: 07-conclusion.Rmd (hunk 1 above)
Comment (pkq) [reply 07-conclusion.Rmd]: Sure. FYI, when I click that link in Firefox, I get a "Your connection is not secure" error. Should it be http://www.moderndive.com?

File: index.Rmd (hunk 2 above)
Comment (pkq) [reply index.Rmd]: Will do, thanks! Not as long as it took you, I'm sure!

File: 07-conclusion.Rmd (hunk 1 above)
Comment (ismayc) [reply 07-conclusion.Rmd]: Sigh, yes. I need to get that switched over to Netlify one of these days to get it secure.

Comment (pkq) [reply 07-conclusion.Rmd]: On it!

**Review by ismayc — APPROVED** [review]

**Comment by pkq** [comment]
Thanks Chester!

**Comment by ismayc** [comment]
You're very welcome! Please do run through Chapters 1-7 of http://moderndive.netlify.com/ as you can. Source is at https://github.com/moderndive/moderndive_book. We'll hopefully be updating the last part of the book this summer.

**Comment by pkq** [comment]
Already forked!

**Pull request merged**